src/Core/Repositories/EntityFramework/Queries/UserBumpAccountRevisionDateByCipherId.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Generic;
+using System.Linq;
+using TableModel = Bit.Core.Models.Table;
+using EfModel = Bit.Core.Models.EntityFramework;
+using Bit.Core.Enums;
+using Bit.Core.Models.Table;
+
+namespace Bit.Core.Repositories.EntityFramework.Queries
+{
+    public class UserBumpAccountRevisionDateByCipherId : IQuery<TableModel.User>
+    {
+        private TableModel.Cipher _cipher { get; set; }
+
+        public UserBumpAccountRevisionDateByCipherId(TableModel.Cipher cipher)
+        {
+            _cipher = cipher;
```

Comment:
Same thing here, `_cipher` should be a `private readonly` field member vs. a property